systemtest/approvals/TestDropUnsampled.approved.json

```diff
@@ -1,4 +1,102 @@
 [
+    {
+        "@timestamp": [
+            "dynamic"
```

Comment:
[**For reviewers**] This change is because PR https://github.com/elastic/apm-tools/pull/61 now sorts the docs after handling `dynamic` fields.